.travis.yml

```diff
@@ -4,8 +4,8 @@ node_js:
 script:
   - yarn lint
   - yarn type-check
-  - yarn test --coverage
   - yarn build
+  - yarn test --coverage
```

Comment:
Putting `test` after `build` makes it throw if there are errors in build files.

Reply:
Humm, it broke coverage

Reply:
@Thomazella

Reply:
oh cool

Reply:
It's considering build files now (testing agains them too isn't a bad idea actually, but not for now).

Reply:
Reverted it.